In [ ]:
from src.googleapi import *

In [2]:
place = find_aggregated_places()


! ERROR: 429


In [3]:
print(place['error']['message'])

The response exceeded the maximum number of allowed places. Only 100 places can be returned in a single response. The request produced 2648 places and therefore no places were returned.


> The response exceeded the maximum number of allowed places. Only 100 places can be returned in a single response. The request produced **2648** places and therefore no places were returned.

# Bounding box

coordinates for copenhagen in a bounding box:

(topleft)
55.82 12.23 
(bottomright)
55.51 12.73

In [12]:
import json

# Opening and reading the JSON file
with open('mini-box.json', 'r') as f:
    # Parsing the JSON file into a Python dictionary
    data = json.load(f)

In [13]:
len(data)

45